### ISSUE TITLE
App Icon Padding having no effect?

### ISSUE THREAD

**Issue by vise890** [home-sweet-gnome/dash-to-panel#429]
I just updated to version 14 and I'm not sure whether this is intended behaviour or not.

The running app indicator used to be flush with the bottom of the panel and the highlight:
![screenshot_1160](https://extensions.gnome.org/extension-data/screenshots/screenshot_1160.png)

This is how `v14` looks like for me:

![v14](https://user-images.githubusercontent.com/1628760/41512118-a6a95122-727b-11e8-90fa-8d3582b79b17.png)

I thought that to make the inner box "flush" with the outer, I'd just reduce the `App Icon Padding` to 0. But that settings seems to have no effect for me.

Am I missing something? Is there anything that can be done to revert to the old look?

I am on Arch with gnome 3.28.2 (on Wayland)

**Comment by charlesg99** [comment]
Hey vise890, does it also happen with a different gnome theme? What is the result if you set a high app icon padding? Thanks!

**Comment by vise890** [comment]
Hi charlesg99,

so yes it *did* happen with the default theme as well and setting the padding to a huge value had no visible effect (in fact i think the screenshot was taken with a padding of 32px and panel size 36px). BUT, yesterday I reverted to `v13` and when I reinstalled `v14` today to double check the answers to your questions, it was fixed :thinking: ...

soo let's close this, I'll re-open an issue if it reappears?

Sorry for the trouble and thank you for the quick response!
